thoth/sourcemanagement/github_authentication.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+# Kebechet
+# Copyright(C) 2020 Sai Sankar Gochhayat
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Handle Github APP Authentication."""
+
+from cryptography.hazmat.backends import default_backend
+import jwt
+import requests
+import time
+import json
+import os
+from pathlib import Path
+
+_BASE_URL = "https://api.github.com/"
+
+
+class GithubAuthentication:
+    """Handles getting the OAuth Token for the Github Application."""
+
+    def __init__(self, slug: str) -> None:
+        """Initialize and check values."""
+        self.slug = slug
+
+        self.GITHUB_PRIVATE_KEY_PATH = str(os.getenv("GITHUB_PRIVATE_KEY_PATH"))
```

Comment:
This will make it an instance attribute, can be lowercase based on pep8.

Reply:
Thanks, I have changed them :)

Reply:
awesome, thanks! :+1: